pkg/drawio/createMapFile.go

```diff
@@ -12,17 +12,15 @@ import (
 var drawioTemplate string
 
 type drawioData struct {
-	FipXOffset int
-	FipYOffset int
-	VSIXOffset int
-	VSIYOffset int
-	VSISize    int
-	RootID     uint
-	IDsPrefix  string
-	// ShowNIIcon says if to display the NI as an NI image, or a VSI image
-	// the rule is that if we have a vsi icon, then we display the NI icon as an NI image
-	ShowNIIcon bool
-	Nodes      []TreeNodeInterface
+	FipXOffset           int
+	FipYOffset           int
+	MiniIconXOffset      int
+	MiniIconYOffset      int
+	MiniIconSize         int
+	RootID               uint
+	IDsPrefix            string
+	canTypeHaveAMiniIcon map[reflect.Type]bool
```

Comment:
Why is this here?
Doesn't it fit better in `styles.go` where all the other maps with `reflect.Type` key are?

Reply:
I rearranged the code, having the styles in a separate struct

Reply:
I'm probably missing something. Why can't `canTypeHaveAMiniIcon` be a global map like `decoreStyles` or the other maps in `styles.go`?

Reply:
// if there are no vsi icon in the canvas, all the ni are displayed as vsi icon, and without mini icons
i.e - if there is no vsi icon, than canTypeHaveAMiniIcon[ni] will be false

Reply:
Got it